lib/Runtime/Library/ScriptFunction.h

```diff
@@ -45,6 +45,7 @@ namespace Js
     public:
         ScriptFunction(FunctionProxy * proxy, ScriptFunctionType* deferredPrototypeType);
         static bool Is(Var func);
+        inline static BOOL Test(JavascriptFunction *func) { return func->GetFunctionInfo()->HasBody(); }
```

Comment:
nit: i would slightly prefer this to be an overload of `Is` rather than a new function